public/nativedisplay.h

```diff
@@ -149,6 +149,20 @@ class NativeDisplay {
 
   virtual void SetExplicitSyncSupport(bool /*explicit_sync_enabled*/) {
   }
+  /**
+  * API to Set the suported modes of the display
+  * @param mode_info vector of drmModeModeInfo
+  */
+  virtual void SetDrmModeInfo(
```

Comment:
You can move this to protected: section, we dont expect it to be called from drmhwc2 or app side.